x11-libs/libva-intel-media-driver/metadata.xml

```diff
@@ -1,7 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<maintainer type="project">
-	<email>media-video@gentoo.org</email>
-</maintainer>
+	<maintainer type="project">
```

Comment:
Want to add yourself?

Reply:
Thanks, I will add as proxied maintainer.